src/util/loadPages.js

```diff
@@ -0,0 +1,51 @@
+const courseList = require('../courses');
+
+const pagelist = [];
+courseList.forEach((course) => {
+  course.pages.forEach((page) => {
+    pagelist.push(page);
+  });
+});
+
+class PageLoader {
+  constructor() {
+    this.pagelist = pagelist;
+  }
+
+  /* get all pages id and type */
+  getPages() {
```

Comment:
According to the API, there does not seem to be a getPages() function

Reply:
Additionally, I would suggest adding the page loading functions in courseloader.js instead of creating a new page loader module

Reply:
I have removed the getPage function and submit new commit.